src/components/PeopleTabel/PersonRow.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+import { PersonLink } from './PersonLink';
+import { useParams } from 'react-router-dom';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonRow: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
```

Comment:
Using `useParams` here is appropriate for determining if the current row should be highlighted. Ensure that the `slug` parameter is correctly passed in the URL to make this feature work as expected.